src/main/resources/templates/application/client/basic-client-details.html

```diff
@@ -11,7 +11,7 @@
     <div id="main-content">
         <form action="#" th:action="@{/application/client/details/basic}" th:object="${clientDetails}" method="post">
             <div th:replace="~{partials/error-summary :: error-summary}"></div>
-            <span class="govuk-caption-l">About your application</span>
+            <span class="govuk-caption-l">About your client</span>
             <h1 class="govuk-heading-l">
                 Basic Client Details
```

Comment:
Should this be sentence case? `Basic client details`?

Reply:
Yeah will amend

Reply:
amended